### PR TITLE
bugfix: add Props folder after running npm i

### PR DESCRIPTION
**Pull request by jugshaurya**
fixes: #96  

**Solution:-**
- added rm -rf scripts, so that netlify build and deploys the apps. (How I know? - checks were not passing on earlier commit because the react-native-elements folder was already there and hence not re-cloning, **see Image below**). Hence added two more execSync Scripts.
Screenshot from 2021-03-25 13-28-03

- Used `execSync` function
 
**Updated-**
Also, readme updated for the same, spaces is because of prettier.

also fixes issue #88 

**Tested How?**
- remove react-native-elements folder and Props folder (Do same if already there)
- run `npm i` after that
- both folders are coming up => Tested.

### CODE REVIEW

**Comment by jugshaurya** [comment]
@pranshuchittora, Please review

File: injectLatestProps.js [hunk 1]
```diff
@@ -1,5 +1,28 @@
-const { exec } = require("child_process");
-exec(
+const { execSync } = require("child_process");
+
+execSync("rm -rf ./react-native-elements/", (err, stdout, stderr) => {
```
Comment (pranshuchittora) [inline injectLatestProps.js:3]: rm -rf is not recommended. Use rimraf

Comment (jugshaurya) [reply injectLatestProps.js:3]: sure.

Comment (jugshaurya) [reply injectLatestProps.js]: @pranshuchittora , added `rimraf` please review.

File: README.md [hunk 2]
```diff
@@ -32,29 +32,33 @@
...
-
 1. Clone the repo
+
 ```sh
 git clone  https://github.com/react-native-elements/playground.git
 cd playground
 ```
-2. Install NPM packages
+
+2. Install NPM packages, add Playground Props, react-native-elements folder
```
Comment (pranshuchittora) [inline README.md:44]: This change is not required

File: injectLatestProps.js [hunk 3]
```diff
@@ -1,5 +1,28 @@
-const { exec } = require("child_process");
-exec(
+const { execSync } = require("child_process");
+
+execSync("rimraf ./react-native-elements/", (err, stdout, stderr) => {
```
Comment (pranshuchittora) [inline injectLatestProps.js:3]: Can you check the dir before deleting?

File: injectLatestProps.js [hunk 4]
```diff
@@ -12,7 +35,9 @@ exec(
     console.log(`stderr: ${stderr}`);
   }
 );
-exec("cp -r ./react-native-elements/website/docs/props ./src/content/Props"),
+
+execSync(
+  "cp -r ./react-native-elements/website/docs/props ./src/content/Props",
```
Comment (pranshuchittora) [inline injectLatestProps.js:40]: This is UNIX based path can you use something like this -> https://www.npmjs.com/package/cp-file

Comment (jugshaurya) [reply injectLatestProps.js:40]: > This is UNIX based path can you use something like this -> https://www.npmjs.com/package/cp-file

sure

File: README.md (hunk 2 above)
Comment (jugshaurya) [reply README.md:44]: ok

File: injectLatestProps.js (hunk 3 above)
Comment (jugshaurya) [reply injectLatestProps.js:3]: done using find command of linux

**Comment by jugshaurya** [comment]
@pranshuchittora please review
Made requested changes
- checking folders before deleting using find-remove package npm
- copying Props directory to src/content/Props via copy-dir package npm
- adding console log to notify what is happening like shown in image below
Screenshot from 2021-04-05 12-40-57

**Comment by mhimanshu712** [comment]
@pranshuchittora I am having the same issue.
Do you approve these changes?

**Comment by jugshaurya** [comment]
@jamesgeorge007 Please review.

**Comment by flyingcircle** [comment]
You shoulddouble check that this still works. The props folder you copy from no longer exists.

**Comment by jugshaurya** [comment]
> You shoulddouble check that this still works. The props folder you copy from no longer exists.

Ok i will get back to you after i check this.

**Comment by jugshaurya** [comment]
@flyingcircle , changed folder from Props to main. Please check and review .

**Pull request merged**